completions/django.completion.sh

```diff
@@ -58,16 +58,16 @@ _python_django_completion()
 }
 
 # Support for multiple interpreters.
-unset pythons
+unset -v pythons
 if _omb_util_command_exists whereis; then
     python_interpreters=$(whereis python | cut -d " " -f 2-)
     for python in $python_interpreters; do
-        pythons="${pythons} $(basename -- $python)"
+        pythons="${pythons} $(basename -- "$python")"
     done
-    pythons=$(echo $pythons | tr " " "\n" | sort -u | tr "\n" " ")
+    pythons=$(echo "$pythons" | tr " " "\n" | sort -u | tr "\n" " ")
 else
     pythons=python
 fi
 
-complete -F _python_django_completion -o default $pythons
+complete -F _python_django_completion -o default "$pythons"
```

Comment:
I believe we originally expected word splitting here. Another approach is to use an array, but we need to anyway apply word splitting to the result of `whereis python` at some point at least once.

Also, for completeness, we actually need to care about `IFS` and possible pathname expansions. Maybe I could consider adding a utility function `_omb_util_split ARRAY_NAME TEXT` in `lib/utils.sh` to perform the safe splitting.

Reply:
That does appear to fix the word splitting issue